lasagne/layers/merge.py

```diff
@@ -11,6 +17,217 @@
 ]
 
 
+CROP_NONE = None
+CROP_LOWER = 'lower'
+CROP_CENTER = 'center'
+CROP_UPPER = 'upper'
+
+
+def autocrop(inputs, cropping):
+    """
+    Crops the given input arrays.
+
+    Cropping takes a sequence of inputs and crops them per-axis in order to
+    ensure that their sizes are consistent so that they can be combined
+    in an element-wise fashion. If cropping is enabled for a specific axis,
+    the minimum size in that axis of all inputs is computed, and all
+    inputs are cropped to that size.
+
+    The per-axis cropping modes are:
+
+    `CROP_NONE` (`None`): this axis is not cropped, inputs are unchanged in
+    this axis
+
+    `CROP_LOWER` (`'lower'`): inputs are cropped choosing the lower portion
+    in this axis (`a[:crop_size, ...]`)
+
+    `CROP_UPPER` (`'lower'`): inputs are cropped choosing the upper portion
+    in this axis (`a[-crop_size:, ...]`)
+
+    `CROP_CENTER` (`'center'`): inputs are cropped choosing the central
+    portion in this axis (`a[offset:offset+crop_size, ...]` where
+    `offset = (a.shape[0]-crop_size)//2)
+
+    Parameters
+    ----------
+    inputs : the input arrays in the form of a list of Theano expressions
+
+    cropping : a list of cropping modes, one for each axis. If length of
```

Comment:
The first thing after the colon should be the type of variable, the explanation should be on the second line: https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections

E.g.:
```
inputs : list of tensor variables
    The arrays to crop according to the cropping specification.
    All arrays must be of the same dimensionality.
```